index.js

```diff
@@ -1,293 +1,177 @@
 module.exports = {
+	root: true,
 	globals: {
-		'__DEV__': true,
-		'process': true,
-		'require': true,
-		'window': true,
-		'global': true,
-		'document': true,
-		'console': true,
-		'module': true,
-		'expect': true,
-		'clearInterval': false,
-		'clearTimeout': false,
-		'setInterval': false,
-		'setTimeout': false
+		__DEV__: true,
+		process: true,
+		window: true,
+		document: true,
 	},
 	env: {
-		'es6': true, // Enables ES6 globals
-		'jest': true
+		es6: true, // Enables ES6 globals
+		'shared-node-browser': true, // restrict to common globals to preserve isomorphic support
+		commonjs: true
 	},
-	extends: ['eslint:recommended', 'plugin:react/recommended'],
+	extends: ['eslint:recommended', 'plugin:react/recommended', 'plugin:react-hooks/recommended'],
+	ignorePatterns: [
+		'**/node_modules/*',
+		'**/build/*',
+		'**/dist/*',
+		'coverage/*'
```

Comment:
Apparently we should use double star at the end, too.

Reply:
Could you share more info on why? These are these same patterns we used to pass through cli args via `--ignore-pattern` which is handled the same (and seems to execute correctly).

If I understand the patterns themselves, single star would include all dir/files at that level, and since we aren't filtering by file extension, blank 1-level scanning deep should be fine. I suspect we may be able to drop it entirely and just say `**/node_modules`, `**/build`, etc.

Reply:
Just based on comments from ESLint chat.  They always recommend the double stars.